### PR TITLE
Readability

### PR DESCRIPTION
**Pull request by Bystroushaak**
Hi.

I added tests and reformated code a little to improve readability, using PEP8. Also fixed corner case for python2 with Iterable import from collections.

### CODE REVIEW

**Comment by Bystroushaak** [comment]
Fixed the merge conflicts.

**Comment by Visgean** [comment]
In the tests you have a typo, 'fist' should be 'first'...

**Comment by Bystroushaak** [comment]
Yeah.

File: test_odictliteral.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        assert odict[["key", "val"]]
+
+    with pytest.raises(SyntaxError):
+        assert odict[("key", "val")]
+
+
+def test_repr():
+    assert repr(odict()) == "odict()"
+    assert repr(odict["key": "val"]) == 'odict[key: val]'
+    assert repr(odict["k": "v", "x": "y"]) == 'odict[k: v, x: y]'
```
Comment (eric-wieser) [inline test_odictliteral.py:62]: This test is wrong, the output should include quotes
